portfolio/src/main/webapp/scripts/templatemo-script.js

```diff
@@ -61,9 +61,11 @@ async function fetchJson() {
     var commentBox = template.content.querySelector('div');
     var temp;
 
+    //clear the list beforehand
+    document.querySelector('#comment-list').innerHTML = "";
```

Comment:
will this remove "comment-template" as well?

Reply:
It will, but the template is already stored in the variable 'template'. It works but it's not well-designed so I will change it. Thank you for pointing it out!